tests/php/LeftAndMainTest.php

```diff
@@ -91,7 +91,15 @@ public function testLeftAndMainSubclasses()
         $menuItems = LeftAndMain::singleton()->MainMenu(false);
         $this->assertGreaterThan(0, count($menuItems));
 
+        // We don't want to test every menu item, some may have relations to TestOnly DataObjects
+        // that are not loaded into the scope of this unit test class
+        $itemsToTest = ['Pages', 'Files', 'Security'];
+        $tested = 0;
         foreach ($menuItems as $menuItem) {
+            if (!in_array($menuItem->Title, $itemsToTest)) {
```

Comment:
Is `Title` sufficiently static (non-translated) to rely on? Is there something less prone to change that we could use (URL?)?

Reply:
True, this could be translated